distribution/zip/jballerina-tools/build.gradle

```diff
@@ -136,6 +136,7 @@ dependencies {
     langserverLib project(':ls-extensions:partial-parser')
     langserverLib project(':ls-extensions:trigger-service')
     langserverLib project(':ls-extensions:bal-shell-service')
+    langserverLib project(':xml-to-record-converter')
```

Comment:
any reason to deviate from the previous conventions? (we have added the json converter as `:ls-extensions:json-to-record-converter`)

Reply:
yeah, this module is not defines under ls-extensions dir,